tools/rpkg/tests/testthat/test-DBItest.R

```diff
@@ -43,4 +43,6 @@ DBItest::test_all(c(
   "bind_factor",
 
   NULL
-))
+  ),
+  ctx <- DBItest::make_context(new("DBIConnector", .drv = duckdb(), .conn_args = list(dbdir = ":memory:")))
```

Comment:
I had to be explicit about creating the context for DBITest. Not sure why but the default context was causing an error.